lib/voyager-message/src/module.rs

```diff
@@ -269,9 +277,9 @@ pub struct UnexpectedClientTypeError {
 }
 
 #[derive(Debug, Clone, thiserror::Error)]
-#[error("invalid IBC interface: this module provides functionality for IBC interface `{expected}`, but the config specifies `{found}`")]
+#[error("invalid IBC interface: this module provides functionality for IBC interfaces `{expected:?}`, but the config specifies `{found}`")]
```

Comment:
format the expected like this:

`expected = expected.join(",")`

Reply:
lgtm!